src/DotVVM.Framework/package-lock.json

```diff
@@ -0,0 +1,6429 @@
+{
```

Comment:
I dont think that this file should be part of repository

Reply:
Why? It guarantees that everyone uses the same dependencies, which seems especially desirable in a CI setting.
(See https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json.)